docs/Configuring.md

```diff
@@ -506,27 +507,52 @@ When `invalidate-bad-cache-results=true` (default: `true`), Cromwell will invali
 
 Cromwell also accepts [Workflow Options](wf_options/Overview#call-caching-options) to override the cache read/write behavior.  
 
+#### Call cache strategy options
```

Comment:
These probably belong under the `Local filesystem options` section since they won't apply to cloud-hosted files which have their own hashing